infra/network/src/main/java/com/simprints/infra/network/url/BaseUrlProvider.kt

```diff
@@ -1,6 +1,6 @@
 package com.simprints.infra.network.url
 
-internal interface BaseUrlProvider {
+interface BaseUrlProvider {
```

Comment:
Just double checking that the `internal` keyword removal is intentionaal

Reply:
It is required in the dashboard/troubleshooting module for the ping button.